<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WeLLC</class>
 <widget class="QMainWindow" name="WeLLC">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>761</width>
    <height>561</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>761</width>
    <height>561</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="Wells">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QWidget" name="welcom" native="true">
      <property name="minimumSize">
       <size>
        <width>761</width>
        <height>555</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>761</width>
        <height>561</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">
background-color: rgb(0, 39, 59);</string>
      </property>
      <widget class="QLabel" name="label_11">
       <property name="geometry">
        <rect>
         <x>270</x>
         <y>30</y>
         <width>191</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>20</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(138, 138, 103);</string>
       </property>
       <property name="text">
        <string>Welcome To</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="label_12">
       <property name="geometry">
        <rect>
         <x>210</x>
         <y>330</y>
         <width>351</width>
         <height>41</height>
        </rect>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(138, 138, 103);</string>
       </property>
       <property name="text">
        <string>Press Login Button To Sign In Your Accounts</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QPushButton" name="log_in">
       <property name="geometry">
        <rect>
         <x>270</x>
         <y>380</y>
         <width>211</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>15</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
	color: rgb(255, 255, 255);
	border-radius: 10px;
}

QPushButton:hover{
	
	background-color: rgb(92, 138, 138);
}</string>
       </property>
       <property name="text">
        <string>Log In</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>:/icon1/icons/arrow-right-circle.svg</normaloff>:/icon1/icons/arrow-right-circle.svg</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>30</width>
         <height>30</height>
        </size>
       </property>
      </widget>
      <widget class="QPushButton" name="log_in_2">
       <property name="geometry">
        <rect>
         <x>200</x>
         <y>460</y>
         <width>341</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>15</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
	color: rgb(255, 255, 255);
	border-radius: 10px;
}

QPushButton:hover{
	
	background-color: rgb(92, 138, 138);
}</string>
       </property>
       <property name="text">
        <string>Log In to Adminster Account</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>:/icon2/icon_white/user-plus.svg</normaloff>:/icon2/icon_white/user-plus.svg</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>30</width>
         <height>30</height>
        </size>
       </property>
      </widget>
      <widget class="QPushButton" name="log_in_3">
       <property name="geometry">
        <rect>
         <x>670</x>
         <y>520</y>
         <width>81</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>15</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
	color: rgb(255, 255, 255);
	border-radius: 10px;
}

QPushButton:hover{
	
	background-color: rgb(92, 138, 138);
}</string>
       </property>
       <property name="text">
        <string>Close</string>
       </property>
       <property name="iconSize">
        <size>
         <width>30</width>
         <height>30</height>
        </size>
       </property>
      </widget>
      <widget class="QWidget" name="widget" native="true">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>80</y>
         <width>761</width>
         <height>81</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="label_2">
          <property name="styleSheet">
           <string notr="true">color: rgb(0,0,0);
font: 75 16pt &quot;MS Shell Dlg 2&quot;;
background-color: rgb(255, 255, 127);</string>
          </property>
          <property name="text">
           <string>Debre Markos Hospital Manegement System Application</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="../../Documents/Firebase/eshe/eshe.qrc"/>
 </resources>
 <connections/>
</ui>
